orttraining/orttraining/test/python/orttraining_transformer_trainer.py

```diff
@@ -258,13 +338,14 @@ def map_optimizer_attributes(name):
         logger.info("\n\nTraining completed. \n\n")
         return TrainOutput(global_step, tr_loss / global_step)
 
-    def _training_step(
-        self, model: ORTTrainer, inputs: Dict[str, torch.Tensor]
-    ) -> float:
+    # def _training_step(
+    #     self, model: ORTTrainer, inputs: Dict[str, torch.Tensor]
+    # ) -> float:
+    def _training_step(self, model, inputs):
```

Comment:
Is there a reason the type hints are being removed from the `_training_step` signature?

Reply:
Yes, `model` can be either `onnxruntime.capi.training.ORTTrainer` or `onxnruntime.experimental.ORTTrainer`, which are different types.